mlir/CMakeLists.txt

```diff
@@ -1,3 +1,16 @@
+if(NOT DEFINED LLVM_VERSION_MAJOR)
+  set(LLVM_VERSION_MAJOR 16)
```

Comment:
Since I don't have the linux packaging context, a question: Does this need to track the LLVM_VERSION field in `external/llvm-project/`? If not, would it make more sense for these values to track the ROCm version instead? That is, instead of `16.0.0`, would `5.5.0` (or, for the development branch, `6.0.0-git`, say) be better?

Reply:
Or if we were going for ROCm versions, since each "minor" version is potentially ABI-breaking, would `55.0.0`, `60.0.0` and so on be better?

Reply:
@tpkessler Your thoughts?

Reply:
See my comment in the linked issue.